### PR TITLE
Refresh reflections of one dataset through disable and enable flag

### PR DESCRIPTION
**Pull request by viktorgitdev**

### CODE REVIEW

**Comment by viktorgitdev** [comment]
1. added a feature to refresh reflection through reflection api by disabling and enabling of reflection
2. set some methods from public to private in .util.refrsh.py

File: dremio_client/util/refresh.py [hunk 1]
```diff
@@ -9,21 +12,65 @@ def refresh_vds_reflection_by_path(client, path=None):
...
         if (parent['datasetType'] == 'VIRTUAL'):
             datasets.append({"id": parent['id']})
         else:
             client.refresh_pds(client.catalog_item(cid=parent['id'], path=None)['id'])
         break
-    refresh_by_path(client, datasets)
+    _refresh_by_path(client, datasets)
+
+
+def _reenable_reflection(client, refl_json):
```
Comment (rymurr) [inline dremio_client/util/refresh.py]: This should be two methods 'enable' and 'disable' reflection

File: dremio_client/dremio_simple_client.py [hunk 2]
```diff
@@ -423,9 +423,21 @@ def graph(self, cid):
...
     def refresh_vds_reflection_by_path(self, path):
-        """ Refresh the metadata for a given physical dataset
+        """ Refresh the reflection for a given virtual dataset
 
         :param path: list ['space', 'folder', 'vds']
         :return: None
         """
         return refresh_vds_reflection_by_path(self, path)
+
+    def refresh_reflections_of_one_dataset(self, path):
```
Comment (rymurr) [inline dremio_client/dremio_simple_client.py:433]: This is a potentially destructive operation as it temporarily removes a reflection causing a fair chunk of work do be done and slowing down queries in the meantime. I think it should have a [big warning](https://pythonhosted.org/an_example_pypi_project/sphinx.html#paragraph-markup) in the pydoc and perhaps a `logging.warn` when the method is called.

**Review by rymurr — CHANGES_REQUESTED** [review]
Not a huge fan of adding this method however its a common pattern. Small pieces of feedback though,

**Pull request merged**